src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js

```diff
@@ -87,21 +87,33 @@ class WayOfTheCrane extends Analyzer {
 
   on_fightend(){
     if(this._inWotc === true){
-      this._wotcTime += (this.selectedCombatant.fightDuration - this._lastTimeStamp)/1000;
+      this._wotcTime += (this.owner.fight.end_time - this._lastTimeStamp)/1000;
     }
   }
 
   statistic() {
     const arrayOfDamageSpells = Array.from(this.customMap.keys());
+    const fightLength = (this.owner.fight.end_time - this.owner.fight.start_time)/1000;
+    let healing = 0;
+    //let overhealing = 0;//doing both of these so we can easily add more in the future
+    const that = this;
+    arrayOfDamageSpells.forEach(function(spell){
+      healing += (that.customMap.get(spell).healing ||0);
+      //overhealing += (that.customMap.get(spell).overheal || 0);
+    });
     return (
       <AzeritePowerStatistic
         size="flexible"
         tooltip={(
-          <ul>
-          {arrayOfDamageSpells.map(spell => (
-            <li>{spell} did {this.customMap.get(spell).healing} healing, {this.customMap.get(spell).overheal} overhealing in {this.customMap.get(spell).casts/3} casts.</li>
-          ))}
-        </ul>
+          <div>
+            You were in WOTC mode for {this._wotcTime} which was {formatPercentage((this._wotcTime/fightLength))}% of the fight.  You can see all all the raw data from them sexy kicks and punches below.   
+            <ul>
+            {arrayOfDamageSpells.map(spell => (
+             <li>{spell} did {this.customMap.get(spell).healing} healing, {this.customMap.get(spell).overheal} overhealing in {this.customMap.get(spell).casts/3} casts.
```

Comment:
Can we format each of the healing and overhealing numbers here? The output looks a bit raw with just the full number listed.

Using something like `formatNumber()` around the healing and overhealing on this.

Reply:
100%